ruby/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM socrata/base
+MAINTAINER Socrata <sysadmin@socrata.com>
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get --force-yes -fuy install software-properties-common && \
+  DEBIAN_FRONTEND=noninteractive add-apt-repository -y ppa:brightbox/ruby-ng && \
+  DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get install -y ruby2.1 && \
+  DEBIAN_FRONTEND=noninteractive apt-get purge -y --auto-remove software-properties-common && \
+  rm -rf /var/lib/apt/lists/*
+
+# skip installing gem documentation
+RUN echo 'gem: --no-rdoc --no-ri' >> "$HOME/.gemrc"
```

Comment:
Can you combine these two RUNs to avoid unnecessary intermediate images, given they are really part of the same step?  Also, did you double check that $HOME is set?